src/coreclr/src/vm/bundle.cpp

```diff
@@ -69,7 +73,7 @@ BundleFileLocation Bundle::Probe(LPCWSTR path, bool pathIsBundleRelative) const
         if (_wcsnicmp(m_basePath, path, baseLen) == 0)
 #endif // TARGET_UNIX
         {
-            path += baseLen; // m_basePath includes count for DIRECTORY_SEPARATOR_CHAR_W
+            utf8Path += baseLen; // m_basePath includes count for DIRECTORY_SEPARATOR_CHAR_W
```

Comment:
I think this is not right. The length of the utf8 and utf16 strings can differ in case of characters that need two or three bytes to encode utf16 character in utf8. So it seems we should keep the original code here and do the transformation to UTF8 right before calling the m_probe.

Reply:
Ah right, we have bundle probe tests, but I think not the ones with extended characters in path.
I think Windows legs were failing, because MSVC didn't liked `+=` on lvalue of type `SString` (Unix builds were fine, although we have only `SString &operator+= (const SString &s)`).

Reply:
Tried a different approach: 0ee19f9, as ::Skip overloads do not accommodate case-insensitivity.